packages/adapter-netlify/README.md

```diff
@@ -42,13 +42,19 @@ You may build your app using functionality provided directly by SvelteKit withou
 
 ### Using Netlify Redirect Rules
 
+> [Netlify docs here](https://docs.netlify.com/routing/redirects/)
+
+Netlify supports declarative redirects via a `_redirects` file placed in your `publish` directory (usually "`build`", as configured in your `netlify.toml` above. NOT in your project root next to `netlify.toml`).
```

Comment:
the `build` directory is generated by SvelteKit, so I wouldn't think you could put it there. I thought on the last issue it was said to go in `static`?

Reply:
Yes! It is `static/` unless you configured `kit.files.assets` to something else in the [project configuration](https://kit.svelte.dev/docs#configuration-files).

Reply:
Yeah my bad... 

```suggestion
Netlify supports declarative redirects via a `_redirects` file. Place it in the `static/` folder, NOT in your project root next to `netlify.toml`.
```